.circleci/config.yml

```diff
@@ -142,6 +142,13 @@ jobs:
   forge-k8s-test:
     executor: ubuntu-medium
     steps:
+      - run:
+          name: Check Forge killswitch
+          shell: /bin/bash
+          command: |
+            if [ -n "$FORGE_KILLSWITCH" ]; then
+              circleci-agent step halt
+            fi
```

Comment:
Let's flip this to FORGE_ENABLED.  It's a little clearer than KILLSWITCH being enabled.

Reply:
Done